matplotlib2tikz/legend.py

```diff
@@ -193,6 +199,17 @@ def draw_legend(data, obj):
             'legend columns={}'.format(obj._ncol)
             )
 
+    # Set color of lines in legend
+    data['legend colors'] = []
```

Comment:
This way, `'legend colors'` gets reset every for every new legend. Is this intended?